data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(1,'Agumon','2020-02-03',0, TRUE,10.23);
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(2,'Gabumon','2018-11-15',2, TRUE,8.00);
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(3,'Pikachu','2021-01-07',1, FALSE,15.04);
+INSERT INTO animals(id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES(4,'Devimon','2017-05-12',5, TRUE,11.00);
```

Comment:
- Optional
I would recommend not entering the Id manually with every entry. The ID will be generated by the Postgres automatically for you. Imagine where you have to enter hundreds if not thousands of records. It will be very difficult to enter the Id with every record without messing up.